dipy/io/vtk.py

```diff
@@ -40,8 +44,9 @@ def save_polydata(polydata, file_name, binary=False, color_array_name=None):
                           binary=binary, color_array_name=color_array_name)
 
 
-def save_vtk_streamlines(streamlines, filename,
-                         to_lps=True, binary=False):
+def save_vtk_streamlines(streamlines: Union[ArraySequence, list],
```

Comment:
Union[ArraySequence, list], Streamlines instead, you might need to create a type name